LTHPasscodeViewController/LTHPasscodeViewController.h

```diff
@@ -314,6 +314,13 @@
  */
 - (void)showLockScreenWithAnimation:(BOOL)animated withLogout:(BOOL)hasLogout andLogoutTitle:(NSString*)logoutTitle;
 /**
+ @brief				Used for displaying the lock. The passcode view is added directly on the keyWindow.
+ @param superview   The superview where is to be presented, used to measure the center of the view.
+ @param hasLogout   Set to @c YES for a navBar with a Logout button, set to @c NO for no navBar.
+ @param logoutTitle The title of the Logout button.
```

Comment:
IMHO the second is quite right, because the lock is presented within the modal, not covering the whole window: 
![screen shot 2017-06-08 at 17 45 29](https://user-images.githubusercontent.com/28537210/26937711-6bb1e9ce-4c72-11e7-9320-73ee784f0648.png)